src/utils/XhrNetworkInterceptor.ts

```diff
@@ -67,8 +67,7 @@ export default {
       _reset();
       network.url = url;
       network.method = method;
-      // @ts-ignore
-      originalXHROpen.apply(this, arguments);
+      originalXHROpen.apply(this, [method, url]);
```

Comment:
This method actually accepts more arguments than `method` and `url` even though we didn't declare them as parameters in our implementation (since we don't need them), but the user might want to pass these other arguments to the original XHR open method.

We can fix this in 2 ways:
1. Use the rest parameters operator to get all other parameters and pass them in the arguments array using the spread operator. Example:
```js
open(method, url, ...otherArgs) {
  originalXHROpen.apply(this, [method, url, ...otherArgs]);
}
```
2. Declare all the remaining parameters and pass each one in the arguments array. Example:
```js
open(method, url, async, user, password) {
  originalXHROpen.apply(this, [method, url, async, user, password]);
}
```

_Generalize this to all similar cases_.